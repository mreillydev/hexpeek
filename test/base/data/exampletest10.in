7:1C/.1
