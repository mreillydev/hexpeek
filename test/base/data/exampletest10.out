At 11 (10 octets requested, 10 per line, hexadecimal) :
0000000000000011: 11223344 55667788 99aabbcc ddeeff00  ."3DUfw.........
